packages/kamino-sdk/src/services/RaydiumService.ts

```diff
@@ -24,8 +35,22 @@ export class RaydiumService {
   }
 
   async getRaydiumPoolLiquidityDistribution(pool: PublicKey): Promise<LiquidityDistribution> {
-    return (await axios.get<LiquidityDistribution>(`https://api.raydium.io/v2/ammV3/positionLine/${pool.toString()}`))
-      .data;
+    let raydiumLiqDistribution = (
+      await axios.get<RaydiumLiquidityDistribuion>(`https://api.raydium.io/v2/ammV3/positionLine/${pool.toString()}`)
+    ).data;
+
+    let liqDistribution: LiquidityDistribution = {
+      data: [],
+    };
+    raydiumLiqDistribution.data.map((entry) => {
+      const liq: LiquidityForPrice = {
+        price: new Decimal(entry.price),
+        liquidity: new Decimal(entry.liquidity),
+      };
+      liqDistribution.data.push(liq);
```

Comment:
This is a bit of a code smell - it's not ideal to push inside the map() function selector, hard to debug and find issues, it's not what you usually expect.

I would consider moving the push to liqDistribution to another line outside of the selector, or just replace map with forEach loop instead. Both of those are IMO better for this.

Reply:
Same comment for the Orca method also